src/SourceBuild/content/eng/publish.proj

```diff
@@ -25,5 +23,6 @@
       MergedAssetManifestOutputPath="$(MergedAssetManifestOutputPath)"
       VmrBuildNumber="$(BUILD_BUILDNUMBER)" />
   </Target>
-
-</Project>
\ No newline at end of file
+  
+  <Target Name="GetMergedManifestPath" DependsOnTargets="MergeAssetManifests" Returns="$(MergedAssetManifestOutputPath)" />
```

Comment:
We shouldn't expose new entry point targets if not strictly necessary. We should just depend on the Build target which makes `MergeAssetManifests` run. The `MergedAssetManifestOutputPath` property already gets moved out of this project by other PRs that are currently in flight: https://github.com/dotnet/installer/pull/19369